src/main/java/com/github/kaklakariada/fritzbox/TestDriver.java

```diff
@@ -33,75 +27,100 @@
 import com.github.kaklakariada.fritzbox.model.homeautomation.DeviceList;
 import com.github.kaklakariada.fritzbox.model.homeautomation.PowerMeter;
 
+/**
+ * this class is a test Driver for the home automation
+ *
+ */
 public class TestDriver {
-    private final static Logger LOG = LoggerFactory.getLogger(TestDriver.class);
-
-    public static void main(String[] args) throws InterruptedException {
-        final Properties config = readConfig(Paths.get("application.properties"));
-        final String url = config.getProperty("fritzbox.url");
-        final String username = config.getProperty("fritzbox.username", null);
-        final String password = config.getProperty("fritzbox.password");
+  private final static Logger LOG = LoggerFactory.getLogger(TestDriver.class);
 
-        LOG.info("Logging in to '{}' with username '{}'", url, username);
-        final HomeAutomation homeAutomation = HomeAutomation.connect(url, username, password);
+  /**
+   * main class
+   * 
+   * @param args
+   * @throws InterruptedException
```

Comment:
Auto generated comments don't help with understanding the code (only mentioned once)